dev-python/axolotl-curve25519/axolotl-curve25519-0.4.1_p2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver
+
+MY_PN="python-${PN}"
+MY_PV="$(ver_cut 1-3)-$(ver_cut 5)"
```

Comment:
Simpler, and more indicative of what you're actually doing to PV:

    ${PV/_p/-}

Reply:
Done.